dashboard/postgres_replicator/logger_test.go

```diff
@@ -0,0 +1,36 @@
+package transfer
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"testing"
+)
+
+func TestNewLogger(t *testing.T) {
+	tableName := "foobar"
```

Comment:
Generally I would also use names like "testTable" instead of using random-looking strings like "foobar".

Reply:
Changed!